resources/public/api/conf/1.0/application.raml

```diff
@@ -41,6 +41,11 @@ uses:
         taxYear:
           description: 'The tax year the data applies to, for example, 2019-20. The start year and end year must not span two tax years. No gaps are allowed, for example, 2019-21 is not valid.'
           example: "2017-18"
+      /amend-relief-investments:
+        (annotations.group):
+          name: Sample Endpoint
+          description: This endpoint allows a developer to amend relief investments.
```

Comment:
You should probably not just call it a sample endpoint for the relief section, try to group it as a relief investment endpoint.